libs/console.js

```diff
@@ -351,9 +333,26 @@
     info: logAtLevel.bind(null, "info"),
     warn: logAtLevel.bind(null, "warn"),
     error: logAtLevel.bind(null, "error"),
-    print: print,
     profile: typeof console !== "undefined" && console.profile ? console.profile.bind(console) : null,
     profileEnd: typeof console !== "undefined" && console.profileEnd ? console.profileEnd.bind(console) : null,
   };
 
+  window.console.buffer = "";
```

Comment:
> Nit: Could we define these only in native.js?

I tried to do that originally, but it resulted in a performance regression, because of the cross-script calls. That was before we started collating main.html scripts, however, and it no longer causes that regression. So I've done this.